androbot/crud.py

```diff
@@ -15,6 +16,91 @@ def get_users(db: Session, skip: int = 0, limit: int = 100):
     return db.query(models.User).offset(skip).limit(limit).all()
 
 
+def get_tg_users(db: Session, skip: int = 0):
+    return db.query(models.TelegramUser).offset(skip).all()
+
+
+def is_tg_user_already_exist(db: Session, tg_user_id: int):
+    return (
+        db.query(models.TelegramUser).filter(models.TelegramUser.tg_user_id == tg_user_id).count()
+        == 1
+    )
+
+
+def create_tg_user(db: Session, user: schemas.TelegramUser):
+    db_user = models.TelegramUser(
+        tg_user_id=user.tg_user_id, name=user.name, username=user.username, specialty=user.specialty
+    )
+    db.add(db_user)
+    db.commit()
+    db.refresh(db_user)
+    return db_user
+
+
+def add_answer(db: Session, answer: schemas.Answer):
+    db_answer = models.Answer(
+        answer_id=answer.answer_id,
+        quest_id=answer.quest_id,
+        tg_user_id=answer.tg_user_id,
+        answer_type=answer.answer_type,
+        text_answer=answer.text_answer,
+        link_to_audio_answer=answer.link_to_audio_answer,
+    )
+    db.add(db_answer)
+    db.commit()
+    db.refresh(db_answer)
+    return db_answer
+
+
+def add_question(db: Session, question: schemas.Question):
+    db_answer = models.Question(
+        quest_id=question.quest_id,
+        question_type=question.question_type,
+        text_answer=question.text_answer,
+    )
+    db.add(db_answer)
+    db.commit()
+    db.refresh(db_answer)
+    return db_answer
+
+
+def remove_tg_user(db: Session, user: schemas.TelegramUser):
+    db.delete(user)
+    db.commit()
+
+
+def remove_answers(db: Session, tg_user_id: int):
+    db.query(Answer).filter(models.Answer.tg_user_id == tg_user_id).delete()
+    db.commit()
+
+
+def remove_questions(db: Session, specialty: str):
+    db.query(Question).filter(models.Question.question_type == specialty).delete()
+    db.commit()
+
+
+def get_passed_questions(db: Session, tg_user_id: int):
+    return list(
+        map(
+            lambda it: it.quest_id,
+            db.query(models.Answer).filter(models.Answer.tg_user_id == tg_user_id).all(),
+        )
+    )
```

Comment:
В целом, кажется можно и компрехэншеном тут жить
```suggestion
    return [
            it.quest_id for it in db.query(models.Answer).filter(models.Answer.tg_user_id == tg_user_id).all(),
    ]
```